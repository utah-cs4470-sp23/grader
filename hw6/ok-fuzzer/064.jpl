show (! ((831 > (- (- 740))) || (((391 < 351) && true) && (if true then false else true))))

