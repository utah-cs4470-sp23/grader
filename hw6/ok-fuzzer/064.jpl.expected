(ShowCmd
 (UnopExpr
  (BoolType)
  !
  (BinopExpr
   (BoolType)
   (BinopExpr
    (BoolType)
    (IntExpr (IntType) 831)
    >
    (UnopExpr (IntType) - (UnopExpr (IntType) - (IntExpr (IntType) 740))))
   ||
   (BinopExpr
    (BoolType)
    (BinopExpr
     (BoolType)
     (BinopExpr (BoolType) (IntExpr (IntType) 391) < (IntExpr (IntType) 351))
     &&
     (TrueExpr (BoolType)))
    &&
    (IfExpr
     (BoolType)
     (TrueExpr (BoolType))
     (FalseExpr (BoolType))
     (TrueExpr (BoolType)))))))
